operations/resourcecategories.md

```diff
@@ -0,0 +1,129 @@
+| `Client` | string | required | Name and version of the client application. |
+| `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). |
+| `ResourceCategoryIds` | array of string | optional, max 1000 items | Unique identifiers of [Resource categories](#resource-category). |
+| `ServiceIds` | array of string | required, max 1000 items | Unique identifiers of [Services](services.md#service) where the age category belong to. |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the resource categories was updated. |
+
+### Response
+
+```javascript
+{
+    "ResourceCategories":[
+        {
+            "Id": "5c0804f9-d03a-4b13-a57d-b00300781a41",
+            "EnterpriseId": "6b78e729-6271-4765-97f7-b0030078151d",
+            "ServiceId": "9b3a6c54-63aa-4383-b50e-b0030078184b",
+            "IsActive": true,
+            "Type": "Bed",
+            "Names": {
+                "en-US": "Bed 1"
+            },
+            "ShortNames": {
+                "en-US": "Bed 1"
+            },
+            "Descriptions": {},
+            "Ordering": 2,
+            "Capacity": 1,
+            "ExtraCapacity": 0,
+            "ExternalIdentifier": null
+        },
+        {
+            "Id": "47d6b462-35ec-467e-a565-b00300781a41",
+            "EnterpriseId": "6b78e729-6271-4765-97f7-b0030078151d",
+            "ServiceId": "9b3a6c54-63aa-4383-b50e-b0030078184b",
+            "IsActive": true,
+            "Type": "Dorm",
+            "Names": {
+                "en-US": "Dorm 1"
+            },
+            "ShortNames": {
+                "en-US": "Dorm 1"
+            },
+            "Descriptions": {},
+            "Ordering": 1,
+            "Capacity": 4,
+            "ExtraCapacity": 0,
+            "ExternalIdentifier": null
+        }
+    ],
+    "Cursor": "47d6b462-35ec-467e-a565-b00300781a41"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ResourceCategories` | array of [Resource category](#resource-category) | optional | Resource categories of the resources. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest resource category returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older resource categories. |
+
+#### Resource category
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the category. |
+| `EnterpriseId` | string | required | Unique identifier of the [Enterprise](enterprises.md#enterprise). |
+| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) of the resource category. |
+| `IsActive` | bool | required | Whether the resource category is still active. |
+| `Type` | string | required | Type of the category. |
```

Comment:
```suggestion
| `Type` | string | required | [Resource category type](#resource-category-type). |
```

Reply:
@MikeAdamsMews I've just noticed we have a huge inconsistency in this. Sometimes we have it as you have provided and sometimes is part of the type - string.